templates/web/pages/product_edit/product_edit_form_display.tt.js

```diff
@@ -34,3 +34,66 @@
 		\$('#nutrition_data_div').show();
 	}
 });
+
+function show_warning(should_show, nutirent_id, warning_message){
+	if(should_show) {
+		\$('#nutriment_'+nutirent_id).css("background-color", "yellow");
+		\$('#nutriment_question_mark_'+nutirent_id).css("display", "inline-table");
+		\$('#nutriment_sugars_warning_'+nutirent_id).text(warning_message);
+	}else {
+		\$('#nutriment_'+nutirent_id).css("background-color", "white");
+		\$('#nutriment_question_mark_'+nutirent_id).css("display", "none");
+	}
+}
+
+var sugars_value;
+var carbohydrates_value;
+var saturated_fats_value;
+var fat_value;
+
+[% FOREACH nutriment IN nutriments %]
```

Comment:
This works, but it will generate a huge JS file, with the code repeated for all nutrients.

It would be better to create a javascript array that contains all nutrients, and then have a for loop in Javascript, instead of a foreach loop in the template language.